sdk/core/core-lro/src/poller/models.ts

```diff
@@ -70,6 +70,18 @@ export interface CreatePollerOptions<TResponse, TResult, TState> {
   withOperationLocation?: (operationLocation: string) => void;
 }
 
+export interface ErrorModel {
```

Comment:
Can we call this `LroError` or `LongRunningOperationError`

Reply:
Do we want to expose this in the API Surface?

Reply:
Renamed it to `LroError`. I kept it private for now along with the `getError` option because we have a good default for it. I can expose them if there is a customer ask where errors are not located in the top-level object in the body for some reason.